frontend/cypress/integration/featureFiles/sidecar_injection.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Controlling sidecar injection
+    In Istio, at installation it is possible to set a default policy for automatic sidecar
+    injection. In addition to the default policy, automatic sidecar injection can be
+    controlled at namespace level and also at deployment level for more specific control.
+    Kiali should provide the needed controls to override the default policy at namespace
+    and deployment levels. Annotations are used to override the default policy and
+    Kiali should reflect these annotations.
+
+    Background:
+        Given user is at administrator perspective
+
+    Scenario: Override the default policy for automatic sidecar injection by enabling it in a namespace
+        Given a namespace without override configuration for automatic sidecar injection
+        When I override the default automatic sidecar injection policy in the namespace to enabled
+        Then I should see the override annotation for sidecar injection in the namespace as "enabled"
```

Comment:
A few of these scenarios are sometimes failing when run locally. I think there's some race condition with clicking the kebab and new data coming in. This gif doesn't show much but you can see the test is stalled until I manually click the kebab:
![TestFailure](https://user-images.githubusercontent.com/6226732/166728582-377390c6-6a4b-46dd-a3be-9659887de051.gif)

Reply:
Hmmm... that particular step never failed for me.
Mabe the auto refresh is happening, making the kebab menu to go away?